var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -170,6 +173,7 @@ def cmake_args(self):
             self.define_from_variant("DD4HEP_USE_HEPMC3", "hepmc3"),
             self.define_from_variant("DD4HEP_USE_GEANT4_UNITS", "geant4units"),
             self.define_from_variant("DD4HEP_BUILD_DEBUG", "debug"),
+            self.define_from_variant("BUILD_DOCS", "docs"),
```

Comment:
If you want to explicitly disallow the LaTeX docs, you can use `CMAKE_DISABLE_FIND_PACKAGE_Latex`, https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html, to avoid picking up a system install of LaTeX.

Reply:
I think it's OK as it is. If docs is off then they are not built, if it's on then try and see. This is the behavior in DD4hep, where it will build documentation if told to do so and either Doxygen or LaTeX are found. So if a system installation is available it should be fine to pick that one up.

Reply:
In spack we try hard not to have the build be different depending on whether you have something installed on your system or not (outside of letting spack know about it with externals). If you want to use the system LaTeX, you can and should `spack external find` it. If you don't want to use a system LaTeX, spack should install it. If you don't want to support LaTeX docs, then you should explicitly disable it.